src/settings_type.h

```diff
@@ -382,82 +382,11 @@ struct ScriptSettings {
-	 * of not being perfect anymore.
-	 */
-	uint32 npf_max_search_nodes;
-	uint32 maximum_go_to_depot_penalty;      ///< What is the maximum penalty that may be endured for going to a depot
-
-	uint32 npf_rail_firstred_penalty;        ///< the penalty for when the first signal is red (and it is not an exit or combo signal)
-	uint32 npf_rail_firstred_exit_penalty;   ///< the penalty for when the first signal is red (and it is an exit or combo signal)
-	uint32 npf_rail_lastred_penalty;         ///< the penalty for when the last signal is red
-	uint32 npf_rail_station_penalty;         ///< the penalty for station tiles
-	uint32 npf_rail_slope_penalty;           ///< the penalty for sloping upwards
-	uint32 npf_rail_curve_penalty;           ///< the penalty for curves
-	uint32 npf_rail_depot_reverse_penalty;   ///< the penalty for reversing in depots
-	uint32 npf_rail_pbs_cross_penalty;       ///< the penalty for crossing a reserved rail track
-	uint32 npf_rail_pbs_signal_back_penalty; ///< the penalty for passing a pbs signal from the backside
-	uint32 npf_buoy_penalty;                 ///< the penalty for going over (through) a buoy
-	uint32 npf_water_curve_penalty;          ///< the penalty for curves
-	uint32 npf_road_curve_penalty;           ///< the penalty for curves
-	uint32 npf_crossing_penalty;             ///< the penalty for level crossings
-	uint32 npf_road_drive_through_penalty;   ///< the penalty for going through a drive-through road stop
-	uint32 npf_road_dt_occupied_penalty;     ///< the penalty multiplied by the fill percentage of a drive-through road stop
-	uint32 npf_road_bay_occupied_penalty;    ///< the penalty multiplied by the fill percentage of a road bay
-};
-
-/** Settings related to the yet another pathfinder. */
-struct YAPFSettings {
-	bool   disable_node_optimization;        ///< whether to use exit-dir instead of trackdir in node key
-	uint32 max_search_nodes;                 ///< stop path-finding when this number of nodes visited
-	uint32 maximum_go_to_depot_penalty;      ///< What is the maximum penalty that may be endured for going to a depot
-	bool   ship_use_yapf;                    ///< use YAPF for ships
-	bool   road_use_yapf;                    ///< use YAPF for road
-	bool   rail_use_yapf;                    ///< use YAPF for rail
-	uint32 road_slope_penalty;               ///< penalty for up-hill slope
-	uint32 road_curve_penalty;               ///< penalty for curves
-	uint32 road_crossing_penalty;            ///< penalty for level crossing
-	uint32 road_stop_penalty;                ///< penalty for going through a drive-through road stop
-	uint32 road_stop_occupied_penalty;       ///< penalty multiplied by the fill percentage of a drive-through road stop
-	uint32 road_stop_bay_occupied_penalty;   ///< penalty multiplied by the fill percentage of a road bay
-	bool   rail_firstred_twoway_eol;         ///< treat first red two-way signal as dead end
```

Comment:
This setting should probably be kept. It's not a pathfinder setting, but rather a "remove two-way signals from the game and replace them with a magic pathfinder hint". These days defaults to "remove two-way signals, since noone used them anyway". In an ideal world this would not have been a pathfinder setting, but a different signal type.